editor/editor_themes.cpp

```diff
@@ -949,7 +949,7 @@ Ref<Theme> create_editor_theme(const Ref<Theme> p_theme) {
 	toolbar_stylebox->set_border_color(accent_color);
 	toolbar_stylebox->set_border_width(SIDE_BOTTOM, Math::round(2 * EDSCALE));
 	toolbar_stylebox->set_content_margin(SIDE_BOTTOM, 0);
-	toolbar_stylebox->set_expand_margin_all(2 * EDSCALE);
+	toolbar_stylebox->set_expand_margin_individual(4 * EDSCALE, 2 * EDSCALE, 4 * EDSCALE, 4 * EDSCALE);
```

Comment:
Why this change?

Reply:
That stylebox is actually clipping buttons, but it's not noticeable if you don't have styles rendered on them. And the sides were too close with the pressed style on, so I extended them a bit.

Reply:
![image](https://github.com/godotengine/godot/assets/2223172/cc3d3780-aee6-4249-a52e-dbdead1e90c9)
Without this change, the side margins match the top margin (bottom does look bad though).